api/src/main/java/org/openmrs/api/UserService.java

```diff
@@ -520,4 +531,15 @@
 	 */
 	@Authorized
 	public User saveUserProperties(Map<String, String> properties);
+	
+	/**
+	 * Change user password by providing answer to secret question
```

Comment:
I think this should be "Change user password given the answer to the secret question"